spec/assets/manifests-repo/deployments/foo.yml

```diff
@@ -3,3 +3,4 @@ name: foo
 templates:
   - foo/bar.yml
 meta: {}
+stub_value: <%= p('stub.value') %>
```

Comment:
I disagree with you. In fact you put main stubs to `meta` section. This `meta` section already has all data that you don't want to place publicly, this means you don't want to push deployment to the repo. The idea of workspace should have user stubs that will never go to any repo.